cmd/litcli/ln.go

```diff
@@ -216,6 +216,13 @@ func (w *resultStreamWrapper) Recv() (*lnrpc.Payment, error) {
 		amountMsat := lnwire.MilliSatoshi(w.amountMsat)
 		milliSatsFP := rfqmath.MilliSatoshiToUnits(amountMsat, *rate)
 		numUnits := milliSatsFP.ScaleTo(0).ToUint64()
+		if numUnits == 0 {
+			return nil, fmt.Errorf("asset rate %v not sufficient "+
```

Comment:
I think we should tune this error message slightly to make it clearer why the payment can't succeed. 

What about something like: 
> smallest payment with asset rate %v is %v, cannot send %v

Reply:
Also I thought the `min_amt` on the RFQ layer was meant to compute this value ahead of time to allow callers to exit sooner before even calling into `lnd`. cc @ffranr

Reply:
should I just derive the smallest payment directly from the asset rate of the quote?